admin/ops/update_json_schemas.py

```diff
@@ -2,37 +2,55 @@
 from dcpy.utils import s3
 from dcpy.models.product import metadata as product_metadata
 from dcpy.models.product.dataset import metadata_v2 as dataset_metadata
+from dcpy.models.lifecycle import ingest as ingest_models
 
 import json
 from tempfile import TemporaryDirectory
 from pathlib import Path
 
+RECIPES_BUCKET = "edm-recipes"
```

Comment:
Curious your rationale on adding a new bucket vs having all our schemas colocated. Just thinking that if we keep adding schemas, not sure they'll be 1-1 with buckets.

Reply:
Yeah, I agree with that, it just on one level made sense to have them in the same place as all the objects that refer to them. But I'm with you - this should really just live in one place

Reply:
And maybe we move it somewhere more "neutral" but that place can be `publishing` for now

Reply:
fixed in new commit